ext/mixed/js/environment.js

```diff
@@ -32,17 +32,36 @@ class Environment {
 
     async _loadEnvironmentInfo() {
         const browser = await this._getBrowser();
-        const platform = await new Promise((resolve) => chrome.runtime.getPlatformInfo(resolve));
-        const modifierInfo = this._getModifierInfo(browser, platform.os);
+        const os = await this._getOperatingSystem();
+        const modifierInfo = this._getModifierInfo(browser, os);
         return {
             browser,
-            platform: {
-                os: platform.os
-            },
+            platform: {os},
             modifiers: modifierInfo
         };
     }
 
+    async _getOperatingSystem() {
+        try {
+            return await this._getPlatformInfo();
```

Comment:
This is an object, but we only need the `os` property if it exists

Reply:
9fad3029d093d9bef210b1436c985e9f9be32ebc